core/src/main/java/nl/nn/adapterframework/http/rest/ApiMemcached.java

```diff
@@ -40,6 +40,7 @@ public class ApiMemcached implements IApiCache {
 
 	protected Logger log = LogUtil.getLogger(this);
 	private MemcachedClient client = null;
+	private final int DEFAULT_OPERATION_TIMEOUT = 10000;
```

Comment:
Als memcached 2.5 seconden aanhoud dan moeten wij dat denk ik ook doen.

```suggestion
	private final int DEFAULT_OPERATION_TIMEOUT = 2500;
```

Reply:
Ik vind het prima, maar waarom is het dan initieel op 10 gezet?